# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/marionette/reftest.xul
# Commit: 96da9d241051
# Full Hash: 96da9d241051d222bdd693bdb519d058a013b59c
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-23 09:46:24
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 8cefa694f811 -r 96da9d241051 testing/marionette/reftest.xul
--- a/testing/marionette/reftest.xul	Fri Mar 22 18:28:31 2019 +0000
+++ b/testing/marionette/reftest.xul	Fri Mar 22 18:28:42 2019 +0000
@@ -1,5 +1,6 @@
 <window id="reftest"
         xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
         hidechrome="true"
-        style="background-color:white; overflow:hidden">
+        style="background-color:white; overflow:hidden"
+        renderroot="content">
 </window>
\ No newline at end of file